### ISSUE TITLE
Shoes manual error

### ISSUE THREAD

**Issue by dredknight** [Shoes3/shoes3#293]
Not a biggie but...
When I search the word "progress" in the manual and then click on the PROGRESS header, the pane part of the manual stops updating and the error below appears.
![progress](https://cloud.githubusercontent.com/assets/12410314/20458145/066f06d4-aea5-11e6-8d75-49adbde75b1d.png)

```Ruby
Error in <unknown> line 0 | 2016-11-19 22:08:11 +0200
bad arguments
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:48:in `eval'
(eval):1:in `block in dewikify_p'
(eval):1:in `call'
(eval):1:in `stack'
(eval):1:in `dewikify_p'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:48:in `eval'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:48:in `dewikify_p'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:115:in `block (2 levels) in dewikify'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:81:in `wiki_tokens'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:113:in `block in dewikify'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:380:in `instance_eval'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:380:in `block in open_methods'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:378:in `call'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:378:in `clear'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:378:in `open_methods'
C:/Program Files (x86)/Shoes/lib/shoes/help.rb:283:in `block (7 levels) in show_search'
-e:1:in `call'
```

**Comment by ccoupe** [comment]
I can re-create the bug this way (Windows and Linux - haven't tried OSX).
1. open the manual and the shoes console (alt-/)
2. Use the search, type in something  that shows a Sub-Section - like edit_line
3. click it. - Check that console.

Of interest, when I Art is the search and select Art for Slots I got 
```ruby
Error in <unknown> line 0 | 2016-11-19 19:02:02 -0700
(eval):1: syntax error, unexpected tCONSTANT, expecting ')'
...ntrol than ", link("strong("Art", :fill => yellow), ".oval")...
...                               ^
(eval):1: syntax error, unexpected ')', expecting end-of-input
...ow), ".oval") { open_link("",") }, ", by offering the ", cod...
...                               ^
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:48:in `eval'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:48:in `dewikify_p'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:128:in `block (2 levels) in dewikify'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:105:in `block in wiki_tokens'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:83:in `map'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:83:in `wiki_tokens'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:113:in `block in dewikify'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:388:in `instance_eval'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:388:in `block (2 levels) in open_methods'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:382:in `each'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:382:in `block in open_methods'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:378:in `call'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:378:in `clear'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:378:in `open_methods'
/home/ccoupe/Projects/shoes3/dist/lib/shoes/help.rb:283:in `block (7 levels) in show_search'
-e:1:in `call'
```
Which suggests it could be badly formed markup (no surprise there). Further, if you try in with Appmethods which is further down in the markup - no problem.  I was going to work on that in the next day or for the plot stuff.

**Comment by ccoupe** [comment]
**And**, after entering the text for the radar charts into the manual and creating a new beta,  this  problem is gone (or moved to somewhere else in the manual). Sigh.

**Comment by ccoupe** [comment]
Closed in Shoes 3.3.2

**Comment by ccoupe** [comment]
Closed in Shoes 3.3.2
